lambdas/service/app.py

```diff
@@ -1580,7 +1581,7 @@ def amend_manifest_location(manifest: Manifest, token: str = None):
         raise BadRequestError(e.args)
     else:
         if time == 0:
-            return amend_manifest_location(manifest=manifest, token=token)
+            return 0, amend_manifest_location(manifest, token)
         else:
             return time, manifest
```

Comment:
There was another suggestion for this part. Since there's no reply, I presume you didn't see it.
https://github.com/DataBiosphere/azul/pull/2914#discussion_r606077541

Reply:
Thank you, I did miss that.